src/improvements/template/DeputyPauseRotationKey.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {VmSafe} from "forge-std/Vm.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+import {L2TaskBase} from "src/improvements/tasks/types/L2TaskBase.sol";
+import {SuperchainAddressRegistry} from "src/improvements/SuperchainAddressRegistry.sol";
+
+import {IDeputyPauseModule} from "lib/optimism/packages/contracts-bedrock/interfaces/safe/IDeputyPauseModule.sol";
+
+import {console} from "forge-std/console.sol";
+
+/// @title DeputyPauseRotationKey
+contract DeputyPauseRotationKey is L2TaskBase {
```

Comment:
This should probably use SimpleTaskBase because it doesn't apply to chains in the superchain, it applies to standalone safes. cc @blmalone to confirm
```suggestion
contract DeputyPauseRotationKey is SimpleTaskBase {
```

Reply:
I concur, if it's superchain level please use SimpleTaskBase.